src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -632,6 +632,17 @@ const ContextMenuActions: ContextMenuAction[] = [
         },
         getDescription: () => {},
     },
+    {
+        isAnonymousAction: true,
+        textTranslateKey: 'reportActionContextMenu.debug',
```

Comment:
We already have this translation 'debug.debug'. Let's reuse it instead of creating a new one.